Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -377,20 +375,38 @@ private function handleRebaseWorkspace(
             $workspace->currentContentStreamId
         );
 
+        // 2) we need to rebase the workspace so that the commands are written to the proper content stream
+        // @todo this should rather be a "Rebase was started"
+
+        $streamName = WorkspaceEventStreamName::fromWorkspaceName($command->workspaceName)->getEventStreamName();
+
+        $this->eventPersister->publishEvents(
+            new EventsToPublish(
+                $streamName,
+                Events::with(
+                    new WorkspaceWasRebased(
+                        $command->workspaceName,
+                        $rebasedContentStream,
+                        $workspace->currentContentStreamId,
+                    ),
+                ),
+                ExpectedVersion::ANY()
+            )
+        );
+
```

Comment:
$this->withContentStreamId($newContentStreamId, function () {
  $originalCommands = $this->extractCommandsFromContentStreamMetadata($workspaceContentStreamName)
  $this->applyOriginalCommands($originalCommands);
});

Reply:
@nezaniel i think you have some local changes lying around that fix this? 

Feel free to force push, i can cherry-pick my only _real_ change on top: https://github.com/neos/neos-development-collection/pull/4708/commits/7a4877caa83855054fd3cefb51de44ffc8e3328f